### ISSUE TITLE
Allow infoboxes to begin stacking to the left of the current column when using vertical infobox stacking.

### ISSUE THREAD

**Issue by Odell74** [runelite/runelite#9893]
If you have the infobox wrap count at say 5, then after 5 infoboxes are shown, the 6th one will appear to the right of the current column. It would be very helpful for the customization that I prefer if you were able to make the infoboxes begin stacking new columns to the left instead, via a checkbox or something similar, below the "display infoboxes vertically" checkbox.

**Comment by G-lint** [comment]
reviving this 7 year old thread to say I'd love this also.

I'd say at least 50% of the players tend to have infoboxes on either the bottom or right side of the their client and having to line things up with the current system is a bit of a pain. Would love something like this to be implemented as a shift+rightclick option on particular groups! 👍 

<img width="340" height="564" alt="Image" src="https://github.com/user-attachments/assets/1f514242-15bf-4bf6-a1ff-9815b9271c90" />

**Comment by RufusSwink** [comment]
Agreed, I love having my boosts just to the left of my inventory but currently the way they stack is a huge pain. Being able to choose if they will stack above or below existing boxes and then if they will wrap to the left or right would be a huge improvement.
